tools/development/parser/meson.build

```diff
@@ -0,0 +1,71 @@
+cc = meson.get_compiler('c')
```

Comment:
Do not need to declare project?
```
$ meson build
The Meson build system
Version: 0.50.1
Source dir: /home/yongjoo/nnstreamer/tools/development/parser
Build dir: /home/yongjoo/nnstreamer/tools/development/parser/build
Build type: native build

ERROR: First statement must be a call to project
```

Reply:
In the end, it will be built along with all other "nnstreamer utilities".
Because the code conversion from gst-parse to nnstreamer-parse is not completed, it won't be able to be built anyway.